content/en/docs/concepts/services-networking/service.md

```diff
@@ -91,23 +110,29 @@ spec:
       targetPort: 9376
 ```
 
-This specification creates a new Service object named "my-service", which
+That manifest creates a new Service named "my-service", which
 targets TCP port 9376 on any Pod with the `app=MyApp` label.
 
 Kubernetes assigns this Service an IP address (sometimes called the "cluster IP"),
 which is used by the Service proxies
 (see [Virtual IPs and service proxies](#virtual-ips-and-service-proxies) below).
 
-The controller for the Service selector continuously scans for Pods that
-match its selector, and then POSTs any updates to an Endpoint object
-also named "my-service".
+The {{<glossary_tooltip text="controller" term_id="controller">}} for the Service
+continuously scans for Pods that match its selector, and then POSTs any updates to
+a set of [EndpointSlices](#endpointslices) that together represent the backend Pods.
+
+The Service controller also manages an older API, [Endpoints](#endpoints), that is
+similar to EndpointSlice (but has some limitations around scalability). The Endpoints
+API is widely used and remains a supported Kubernetes API.
```

Comment:
for correctness,  there is no services controllers , there are a `endpoints controller` and a `endpointslices controller` , there is also an additional `endpointslice mirroring controller` that mirrors endpoints to endpointslices for compatibility, but I think that this last bit is more an implementation detail that users may not need

Reply:
I might change “Service controller” to “control plane”.

Reply:
yeah, if you want to be more specific you can say is the `controller-manager` component, that is the one that hosts those controllers